prow/test.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: v1
```

Comment:
What is this file?

Reply:
random test file, shouldn't be here. Deleting